CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ A breaking change will get clearly notified in this log.
 
 ## In `master`
 
+- _Warning_ Calling TransactionBuilder without a `fee` param is now deprecated
+  and will issue a warning. In a later release, it throw an error.
```

Comment:
In a later release, it **will** throw an error.

Reply:
Fixed!